diff --git a/node_modules/glightbox/dist/js/glightbox.js b/node_modules/glightbox/dist/js/glightbox.js
index 678f7cb..d826f42 100644
--- a/node_modules/glightbox/dist/js/glightbox.js
+++ b/node_modules/glightbox/dist/js/glightbox.js
@@ -2184,15 +2184,6 @@
         this.build();
         animateElement(this.overlay, this.settings.openEffect === 'none' ? 'none' : this.settings.cssEfects.fade["in"]);
         var body = document.body;
-        var scrollBar = window.innerWidth - document.documentElement.clientWidth;
-        if (scrollBar > 0) {
-          var styleSheet = document.createElement('style');
-          styleSheet.type = 'text/css';
-          styleSheet.className = 'gcss-styles';
-          styleSheet.innerText = ".gscrollbar-fixer {margin-right: ".concat(scrollBar, "px}");
-          document.head.appendChild(styleSheet);
-          addClass(body, 'gscrollbar-fixer');
-        }
         addClass(body, 'glightbox-open');
         addClass(html, 'glightbox-open');
         if (isMobile$1) {
@@ -2999,7 +2990,7 @@
           }
           var body = document.body;
           removeClass(html, 'glightbox-open');
-          removeClass(body, 'glightbox-open touching gdesc-open glightbox-touch glightbox-mobile gscrollbar-fixer');
+          removeClass(body, 'glightbox-open touching gdesc-open glightbox-touch glightbox-mobile');
           _this8.modal.parentNode.removeChild(_this8.modal);
           _this8.trigger('close');
           if (isFunction(_this8.settings.onClose)) {
